man/eza_colors-explanation.5.md

```diff
@@ -1,6 +1,6 @@
 # Name
 
-eza_colors-explanation — more details on customizing eza colors
+eza_colors-explanation — more details on customizing eza colours
```

Comment:
Color looks so weird without a `u` 😬 

I'm just doing a random drive-by over the diff and caught this. I assume you want to normalize on `color`, this isn't the only instance where you've changed from `color` to `colour`, so you'd want to have another pass to correct that 😅 

Looks like any actual code is fine and intentional, same with a test, so it's only in docs/comments that about 5-ish instances were missed (_Unless that's also intentional, but you've also got comments with `color` too_ 🤷‍♂️ ).

Reply:
well no, i wanted to normalize the color only in the code... I tried to go back and change all the instances of the comments